### PR TITLE
replace feast android sqs queue to follow convention in repo

### PR DESCRIPTION
**Pull request by ecguardian**
## What does this change?
This PR renames the `feast android` with `feast google` to align with the project's naming conventions. This change improves code consistency

- `FeastAndroidSubscriptionsQueue` to `FeastGoogleSubscriptionsQueue`
- `feast-android-subscriptions-to-fetch` to  `feast-google-subscriptions-to-fetch`

## How to test

This has been tested in CODE, making sure that the upstream and downstream lambdas work

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: cloudformation.yaml [hunk 1]
```diff
@@ -772,26 +772,26 @@ Resources:
...
-      QueueName: !Sub ${App}-${Stage}-feast-android-subscriptions-to-fetch
+      QueueName: !Sub ${App}-${Stage}-feast-google-subscriptions-to-fetch
       RedrivePolicy:
-        deadLetterTargetArn: !GetAtt FeastAndroidSubscriptionsQueueDlq.Arn
+        deadLetterTargetArn: !GetAtt FeastGoogleSubscriptionsQueueDlq.Arn
         maxReceiveCount: 8
       KmsMasterKeyId: alias/aws/sqs
       Tags:
-        - Key: Stage
+        - Key: Stag
```
Comment (tjmw) [inline cloudformation.yaml:784]: I think there's a typo here?

Comment (ecguardian) [reply cloudformation.yaml]: yes, you are right! thanks

**Review by rBangay — APPROVED** [review]
👍

**Review by MaelGNM — APPROVED** [review]
LGTM!

**Pull request merged**